docs/contributrs-quick-start.rst

```diff
@@ -0,0 +1,915 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+*************************
+Contributor's Quick Guide
+*************************
+
+.. contents:: :local:
+
+
+Prerequisites
+#############
+
+1. Docker Community Edition
+2. Docker Compose
+3. pyenv (you can also use pyenv-virtualenv or virtualenvwrapper)
```

Comment:
```suggestion
* Docker Community Edition (Linux)
* Docker Desktop (Windows/Mac OS)
* Docker Compose
* pyenv (you can also use pyenv-virtualenv or virtualenvwrapper)
```

Reply:
Can you also add docs for Windows/Mac OS (not Apple Silicon based 🤣)?

Reply:
I did try to setup on windows and mac in VM but due to docker virtualisation error i could not try. And on windows also i was not able to setup pyenv virtualenv.

![DockerError](https://user-images.githubusercontent.com/11897651/99148486-ef3ecc80-26ad-11eb-804f-024f91a0716f.png)
![WinError](https://user-images.githubusercontent.com/11897651/99148492-f5cd4400-26ad-11eb-8341-4eac7df91a96.png)